generators/upgrade/index.js

```diff
@@ -108,6 +110,7 @@ module.exports = UpgradeGenerator.extend({
                     this.error('bower install failed.');
                 }
             }
+            shelljs.rm('-Rf', `${SERVER_MAIN_RES_DIR}keystore.jks`);
```

Comment:
Yes, it prevents the only merge conflict when upgrading a UAA so looks good to me.  I tested with a default UAA